src/hotspot/share/runtime/globals.hpp

```diff
@@ -1990,6 +1990,13 @@ const int ObjectAlignmentInBytes = 8;
                                                                             \
   product(bool, OMUseC2Cache, true, "")                                     \
                                                                             \
+  product(bool, OMC2UnrollCacheLookup, false, "")                           \
+                                                                            \
+  product(bool, OMC2UnrollCacheLookupLoopTail, true, "")                    \
+                                                                            \
+  product(int, OMC2UnrollCacheEntires, 8, "")                               \
```

Comment:
There is a typo: Entires -> Entries

Reply:
Also, could perhaps collapse some flags, e.g. treat OMC2UnrollCacheEntries==0 as OMC2UnrollCacheLookup==false, and get rid of the latter.